sdk/textanalytics/ai-text-analytics/test/error.spec.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+
+import { makeTextAnalyticsErrorResult } from "../src/textAnalyticsResult";
+import { ErrorCodeValue, InnerErrorCodeValue } from "../src/generated/models";
+
+/**
+ * Works around an issue with the swagger models and casing
+ */
+function cap<T extends ErrorCodeValue | InnerErrorCodeValue>(code: T): T {
```

Comment:
I'm confused by why this function is needed - is the enum type that is generated by swagger incorrect? If so, it seems like we should fix that rather than working around it?

Reply:
This was related to the issue with the swagger but I've just patched the swagger using the README in the latest commits.